### DISCUSSION TITLE
Can't install Strapi with manual settings (MySQL database)

### DISCUSSION THREAD

**Discussion by teinett** [strapi/strapi#6592] [Legacy Topics]
I am trying to install Strapi following the docs: https://strapi.io/documentation/v3.x/installation/cli.html

I use command: `yarn create strapi-app cms`

The installation process stuck after choosing database (I choose MySQL):
```
yarn create strapi-app cms
yarn create v1.22.4
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
success Installed "create-strapi-app@3.0.1" with binaries:
      - create-strapi-app
[####################################################################################] 84/84Creating a new Strapi application at /Users/XXX/Documents/XXX/cms.

? Choose your installation type Custom (manual settings)
? Choose your default database client mysql
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...

error Command failed.
Exit code: 1
Command: /usr/local/bin/create-strapi-app
Arguments: cms
Directory: /Users/XXX/Documents/XXX
Output:

info Visit https://yarnpkg.com/en/docs/cli/create for documentation about this command.
```

Screenshot:
<img width="1400" alt="2020-06-08 22-28-18" src="https://user-images.githubusercontent.com/1038168/84072193-7d17dc80-a9d7-11ea-8112-b26d0e027ca5.png">

Strapi 3.0.1
Node: 12.18.0
Yarn: 1.22.4

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by derrickmehaffy** [comment]
Can you please clarify what OS you are using?

**Reply by teinett** [reply]
MacOS Catalina 10.15.5 with all recent updates.

**Reply by derrickmehaffy** [reply]
By chance are you using the zsh shell? And if so can you please test using the normal bash shell?

(If you open a terminal and type `/bin/bash` it will open a bash shell)
